app/api/stats.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from flask import jsonify, request, g, abort, url_for, current_app
+from .. import db
+from ..models import Stop
+from . import api
+from sqlalchemy import func 
+
+
+@api.route('/stats')
+def get_stats():
+	stops = db.session.query(func.min(Stop.stop_lat) \
```

Comment:
The result of this query is not a set of stops so maybe the name is confusing. Please change it to `result` or something like that.